github-integration-its/src/main/java/org/jenkinsci/plugins/github_integration/SomeTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jenkinsci.plugins.github_integration;
```

Comment:
If it be under `.github.integration` as base package will be no conflicts i think